src/parser/youtube/History.ts

```diff
@@ -25,4 +26,36 @@ export default class History extends Feed<IBrowseResponse> {
       throw new Error('No continuation data found');
     return new History(this.actions, response, true);
   }
-}
\ No newline at end of file
+
+  /**
+   * Removes a video from watch history.
+   */
+  async removeVideo(video_id: string): Promise<Boolean> {
+    let feedbackToken;
+
+    for (const section of this.sections) {
+      for (const content of section.contents) {
+        const video = content as Video;
+        if (video.id === video_id) {
+          if (video.menu) {
+            feedbackToken = video.menu.top_level_buttons[0].endpoint.payload.feedbackToken;
+          }
+        }
```

Comment:
This is missing a `break` so loops can be interrupted for better performance and clarity.

Additionally, nested `if` can be simplified into a single one.

```suggestion
        if (video.id === video_id && video.menu) {
            feedbackToken = video.menu.top_level_buttons[0].endpoint.payload.feedbackToken;
            break;
        }
```